tests/test_merge_request.py

```diff
@@ -216,11 +216,11 @@ def test_accept_remove_branch(self):
             self.merge_request.accept(remove_branch=boolean)
             self.api.call.assert_called_once_with(PUT(
                 '/projects/1234/merge_requests/54/merge',
-                dict(
-                    merge_when_pipeline_succeeds=True,
-                    should_remove_source_branch=boolean,
-                    sha='badc0de',
-                )
+                {
+                    'merge_when_pipeline_succeeds': True,
+                    'should_remove_source_branch': boolean,
+                    'sha': 'badc0de',
+                }
```

Comment:
I'm not 100% convinced that this is really more readable, but if that's what the linters/tooling decree... :grin: